cpp/CMakeLists.txt

```diff
@@ -503,13 +505,16 @@ target_compile_definitions(cudf PUBLIC "SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_L
 # Compile stringified JIT sources first
 add_dependencies(cudf jitify_preprocess_run)
 
+set_target_properties(nvcomp PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
Could this be moved into the `get_nvcomp.cmake` file? I assume this only applies to when `nvcomp` is not found and we build a static library to static link.

Reply:
(Hi @kkraus14!). I'm hoping to take this out entirely before merging. I'm waiting for nvcomp to add this to their own cmakelists. I added this just to confirm that it would fix CI.